particle/particle/particle.py

```diff
@@ -264,12 +264,13 @@ def S(self):
 
     @property
     def charge(self):
-       return self.three_charge / 3
+        """The particle charge, in units of the positron charge."""
+        return self.three_charge / 3
 
     @property
     def three_charge(self):
-        'The particle charge (integer * 3).'
-        return Charge(self.pdgid.three_charge)
```

Comment:
Wasn't Charge partially used because it could produce nice output, like "+" for 3? Do we still get that easily? And you can write Charge.p instead of 3.

Reply:
I don-t know ;-). But I find it non-ideal to get the following:

```
In [3]: Particle.from_pdgid(111).three_charge
Out[3]: <Charge.o: 0>

In [4]: Particle.from_pdgid(-211).three_charge
Out[4]: <Charge.m: -3>
```

If `charge` gives a float, then `three_charge` should give an int, for consistency. Agree?

Reply:
The IntEnum behaves exactly like an int under duck typing, though.

Reply:
Sure, sure. What annoys me is that the user will get `Charge.m: -3>` and alike. I would prefer to have `Particle.three_charge` consistent with `PDGID.three_charge`, so plain ints. The Enums are really nice for all the conversion and handling machinery you wrote, and work perfectly there.

Reply:
By my same argument, `Particle.from_pdgid(111).three_charge == Charge.o` will work regardless of which one it returns. So I would be okay with it returning a plain int, as I think that's the main reason to have the enum.